### ISSUE TITLE
GUI suggestion

### ISSUE THREAD

**Issue by acontrerasg** [qjiangzhao/TEtrimmer#47]
Hello, 

I have been trying to streamline my manual curation process of TE libraries using your GUI which I think is a very nice idea. 

However, when blasting  the TE model against the genome, I can easily get 1.5k hits or more, which it is an issue when  trying to generate a MSA alignment out of them. 

When running TE trimmer, you have two default options` --top_mas_lines 100` and ` --max_msa_lines 100 ` which  sorts and subsets alignments. 

I wonder if it will be possible to include also this same logic in the GUI when using Other_consensus_lib as main input. 

Thanks!

**Comment by qjiangzhao** [comment]
Hi,

Thanks for your suggestion. I added --top_mas_lines 100 and --max_msa_lines 100 options. You can clone the updated version and try it. 

Yours sincerely
Jiangzhao

**Comment by acontrerasg** [comment]
Thanks a lot! Looking forward to it.

**Comment by acontrerasg** [comment]
hello I am reopening this issue, because I download the new GUI tool, load my custom genome and custom TE library in the GUI, blasted a TE and got 1300 hits. How can I change these settings? Thanks!

**Comment by acontrerasg** [comment]
Hello, thanks for paying attention to this. 
Yes I can see the options in the code, and indeed when I open the .fa file I see only 100 sequences despite the txt having many more. 
So that is working as intended, however i cant find the option in the GUI under "Setting ->  modify Parameters".
See picture attached: 
![image](https://github.com/user-attachments/assets/f77f5755-ac38-49e3-9889-9d502806c95f)

**Comment by qjiangzhao** [comment]
Yes, I haven't integrated those options into the setting dialogue. But will do that soon! Thanks for your suggestion.

**Comment by qjiangzhao** [comment]
I uploaded a new version of TEtrimmerGUI. You can modify those two parameters in the settings menu.

**Comment by acontrerasg** [comment]
Thanks a lot for addressing this!
